src/test/resources/com/puppycrawl/tools/checkstyle/grammars/javadoc/javadocTags/InputJavadocWithCrAsNewline.txt

```diff
@@ -0,0 +1 @@
+* Test case for detecting CR newlines.
* @author Author Name

```

Comment:
@ps-sp It is probably best for your UT that you make an exact duplicate of `InputLeadingAsterisksExtended.txt` but change all `\n` to `\r`.